pkg/clusterlogging/clusterlogforwarder.go

```diff
@@ -35,6 +35,10 @@ func NewClusterLogForwarderBuilder(
 	builder := &ClusterLogForwarderBuilder{
 		apiClient: apiClient,
 		Definition: &clov1.ClusterLogForwarder{
+			TypeMeta: metav1.TypeMeta{
```

Comment:
You can safely remove it.

Reply:
now all is broken on missing GVK if I remove it

Reply:
=== RUN   TestClusterLogForwarderGet
    clusterlogforwarder_test.go:210: 
                Error Trace:    /home/elgerman/eco-gotest/eco-goinfra-ir/pkg/clusterlogging/clusterlogforwarder_test.go:210
                Error:          Not equal: 
                                expected: &v1.ClusterLogForwarder{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ObjectMeta:v1.ObjectMeta{Name:"instance", GenerateName:"", Namespace:"openshift-logging", SelfLink:"", UID:"", ResourceVersion:"999", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, Spec:v1.ClusterLogForwarderSpec{Inputs:[]v1.InputSpec(nil), Outputs:[]v1.OutputSpec(nil), Filters:[]v1.FilterSpec(nil), Pipelines:[]v1.PipelineSpec(nil), ServiceAccountName:"", OutputDefaults:(*v1.OutputDefaults)(nil)}, Status:v1.ClusterLogForwarderStatus{Conditions:status.Conditions(nil), Inputs:v1.NamedConditions(nil), Outputs:v1.NamedConditions(nil), Filters:v1.NamedConditions(nil), Pipelines:v1.NamedConditions(nil)}}
                                actual  : &v1.ClusterLogForwarder{TypeMeta:v1.TypeMeta{Kind:"ClusterLogForwarder", APIVersion:"logging.openshift.io/v1"}, ObjectMeta:v1.ObjectMeta{Name:"instance", GenerateName:"", Namespace:"openshift-logging", SelfLink:"", UID:"", ResourceVersion:"999", Generation:0, CreationTimestamp:time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC), DeletionTimestamp:<nil>, DeletionGracePeriodSeconds:(*int64)(nil), Labels:map[string]string(nil), Annotations:map[string]string(nil), OwnerReferences:[]v1.OwnerReference(nil), Finalizers:[]string(nil), ManagedFields:[]v1.ManagedFieldsEntry(nil)}, Spec:v1.ClusterLogForwarderSpec{Inputs:[]v1.InputSpec(nil), Outputs:[]v1.OutputSpec(nil), Filters:[]v1.FilterSpec(nil), Pipelines:[]v1.PipelineSpec(nil), ServiceAccountName:"", OutputDefaults:(*v1.OutputDefaults)(nil)}, Status:v1.ClusterLogForwarderStatus{Conditions:status.Conditions(nil), Inputs:v1.NamedConditions(nil), Outputs:v1.NamedConditions(nil), Filters:v1.NamedConditions(nil), Pipelines:v1.NamedConditions(nil)}}